Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -59,7 +47,7 @@ public final class DependencyTreeGenerator {
             for try await generatedRoot in taskGroup {
                 generatedRoots.append(generatedRoot)
             }
-            return generatedRoots.sorted().joined(separator: "\n\n")
+            return generatedRoots.filter { !$0.isEmpty }.sorted().joined(separator: "\n\n")
```

Comment:
shouldn't affect much, but I've seen some weird whitespace in the generated files and I think this might fix it.